robber/matchers/exception.py

```diff
@@ -0,0 +1,33 @@
+from robber import expect
+from robber.matchers.base import Base
+
+
+class ExceptionMatcher(Base):
+    """
+    expect(lambda: call_something(with_some_params)).to.throw(any_exception)
+    """
+    @property
+    def _raised(self):
+        raised = None
+
+        if not callable(self.actual):
+            raise TypeError('{actual} is not callable'.format(actual=self.actual))
+
+        try:
+            self.actual()
+        except BaseException as raised:
+            return raised
+
+    def matches(self):
+        return isinstance(self._raised, self.expected)
```

Comment:
I'm thinking about it since my first design, but super() seems more reasonable. We can't say that ChildException is not a ParentException, `sure` approached it the same way I did here.